src/build.cc

```diff
@@ -694,9 +691,9 @@ void RealCommandRunner::Abort() {
 bool RealCommandRunner::CanRunMore() const {
   size_t subproc_number =
       subprocs_.running_.size() + subprocs_.finished_.size();
-  return (int)subproc_number < config_.parallelism
-    && ((subprocs_.running_.empty() || config_.max_load_average <= 0.0f)
-        || GetLoadAverage() < config_.max_load_average);
+  return static_cast<int>(subproc_number) < config_.parallelism &&
```

Comment:
Better to convert config_.parallelism to size_t than to cast subproc_number to int.